pkg/apps/gitops.go

```diff
@@ -128,10 +132,11 @@ func (o *GitOpsOptions) DeleteApp(app string, alias string) error {
 		}
 		return nil
 	}
+	appGitName := strings.TrimPrefix(app, "jx-app-")
```

Comment:
It looks like here you're removing the `jx-app` prefix from the delete command only?

Reply:
Fixed.